sectionsUtilityTool/src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -49,6 +53,16 @@
 	private String canvasToken;
 	private String canvasURL;
 	ResourceBundle props = ResourceBundle.getBundle("sectiontool");
+	/*
+	 * When needed to add another api call add to "apiListRegex" list. Make sure to add a log message with a particular api to the  "apiListDebugMsg" list also and in the same order. 
+	 * Here we are taking the advantage of an ArrayList being ordered list.  
+	 */
+	private static final ArrayList<String> apiListRegex = new ArrayList<String>(Arrays.asList(CANVAS_API_TERMS, CANVAS_API_CROSSLIST,CANVAS_API_RENAME_COURSE, CANVAS_API_GETCOURSE_BY_UNIQNAME,CANVAS_API_GETCOURSE_BY_UNIQNAME_NO_SECTIONS,
+			CANVAS_API_ENROLLMENT, CANVAS_API_GETCOURSE_INFO,CANVAS_API_DECROSSLIST, CANVAS_API_GETSECTION_INFO,CANVAS_API_GETSECTION_PER_COURSE,CANVAS_API_GETALLSECTIONS_PER_COURSE));
+	
+	private static final ArrayList<String> apiListDebugMsg = new ArrayList<String>(Arrays.asList("The canvas api request for terms","The canvas api request for crosslist","The canvas api request for rename a course","The canvas api request for getting courses by uniqname",
+			"The canvas api request for getting courses by uniqname not including sections","The canvas api request for enrollment","The canvas api request for getting course info","The canvas api request for decrosslist",
+			"The canvas api request for getting section info","The canvas api request for getting section info for a given course","The canvas api request for getting all section info for a given course"));
 
```

Comment:
Why not use a hashmap to associate the API regex symbols to their descriptions?  Also, "The canvas api request for" is repeated a lot. Remove it from the strings to make it easier to read the code.  When the strings are displayed, prefix that phrase.